pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -310,23 +310,21 @@ func NewContainerManager(mountUtil mount.Interface, cadvisorInterface cadvisor.I
 	}
 
 	// Initialize CPU manager
-	if utilfeature.DefaultFeatureGate.Enabled(kubefeatures.CPUManager) {
-		cm.cpuManager, err = cpumanager.NewManager(
-			nodeConfig.ExperimentalCPUManagerPolicy,
-			nodeConfig.ExperimentalCPUManagerPolicyOptions,
-			nodeConfig.ExperimentalCPUManagerReconcilePeriod,
-			machineInfo,
-			nodeConfig.NodeAllocatableConfig.ReservedSystemCPUs,
-			cm.GetNodeAllocatableReservation(),
-			nodeConfig.KubeletRootDir,
-			cm.topologyManager,
-		)
-		if err != nil {
-			klog.ErrorS(err, "Failed to initialize cpu manager")
-			return nil, err
-		}
-		cm.topologyManager.AddHintProvider(cm.cpuManager)
+	cm.cpuManager, err = cpumanager.NewManager(
+		nodeConfig.ExperimentalCPUManagerPolicy,
+		nodeConfig.ExperimentalCPUManagerPolicyOptions,
+		nodeConfig.ExperimentalCPUManagerReconcilePeriod,
```

Comment:
Should these variables still be prefixed with `Experimental` with this going to GA?

Reply:
Good point. Probably not. I'll push a separate commit to do the trivial rename